server/src/test/java/io/crate/integrationtests/CopyFromFailFastITest.java

```diff
@@ -91,7 +91,7 @@ public void test_copy_from_with_fail_fast_with_single_node() throws Exception {
                 "COPY t FROM ? WITH (bulk_size = 1, fail_fast = true)", // fail_fast = true
                 new Object[]{target.toUri() + "*"}))
             .isExactlyInstanceOf(JobKilledException.class)
-            .hasMessageContaining("ERRORS: {Cannot cast value `fail here` to type `integer`");
+            .hasMessageContaining("ERRORS: {For input string: \"fail here\"");
```

Comment:
I found new message not that nice - its because in old code path (`legacyInsert`), we manually raised `ConversionException` in `ValidatedRawInsertSource.parse`. Now we are taking default JDK `NumberFormatException` from `implicitCast`. 

Should I wrap `impicitCast` of all types into try and raise `ConversionException`?